NerlNet_subnets_config

```diff
@@ -0,0 +1,5 @@
+# List of relevant subnets that NerlNet will use. Instead of 192.168.x.x write 192.168.
+
+192.168.
+10.0.
+192.168.100.
```

Comment:
Remove 192.168.100
It was only for example 🙂

Reply:
done